src/components/atoms/UiBackdrop/UiBackdrop.stories.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { UiBackdrop } from '@/../index';
```

Comment:
@ArkadiuszKrzymienski WDYT about this approach to imports? IMO we can import components like that
```
import  { 
UiDropdown, 
UiButton, 
UiText, 
UiInput, 
UiPopover, 
UiBulletPoints, 
UiMessage
} from '@/../index';
import UiDropdownItem from './_internal/UiDropdownItem.vue';
```
instead
```
import UiDropdown from '@/components/molecules/UiDropdown/UiDropdown.vue';
import UiDropdownItem from '@/components/molecules/UiDropdown/_internal/UiDropdownItem.vue';
import UiButton from '@/components/atoms/UiButton/UiButton.vue';
import UiText from '@/components/atoms/UiText/UiText.vue';
import UiInput from '@/components/atoms/UiInput/UiInput.vue';
import UiPopover from '@/components/molecules/UiPopover/UiPopover.vue';
import UiBulletPoints from '@/components/molecules/UiBulletPoints/UiBulletPoints.vue';
import UiMessage from '@/components/templates/UiMessage/UiMessage.vue';
```

Reply:
great idea 👍 maybe we should add some alias to the index file e.g. `@index`? I also suggest importing types like this 🤔 WDYT?
```
import type { CheckboxProps, IconProps } from '@/../index';
```

instead

```
import type { CheckboxProps } from '@/components/atoms/UiCheckbox/UiCheckbox.vue';
import type { IconProps } from '@/components/atoms/UiIcon/UiIcon.vue';
```